nucypher/network/server.py

```diff
@@ -131,7 +130,7 @@ def ping():
         """
 
         try:
-            requesting_ursula = Ursula.from_bytes(request.data, registry=this_node.registry)
+            requesting_ursula = Ursula.from_bytes(request.data)
```

Comment:
:+1: - This has been disabled for now but - how does a stranger get verified against the registry if not injected here?

Reply:
First let me say: this change is utterly extemporaneous; I did it only to get a green check (this test had apparently been failing for a while, but went unnoticed because we weren't running this module).

Second: the simplest vision I can think to express is, I guess:

```
requesting_ursula = Ursula.from_bytes(request.data)
requesting_ursula.mature()
requesting_ursula.verify_node(registry=this_node.registry)
```

...though in calmer times, I might also even ask why verification is important here.